app/scripts/app.js

```diff
@@ -514,120 +501,3 @@ angular
     }
   });
 
-hawtioPluginLoader.addModule('openshiftConsole');
```

Comment:
Missed this earlier, I'm actually surprised you were able to remove this.  How is the openshiftConsole module getting registered with hawtioPluginLoader now?